src/main/interface/cli.c

```diff
@@ -668,6 +703,34 @@ static void cliSetVar(const clivalue_t *var, const int16_t value)
     }
 }
 
+
+static void cliOSDProfileSetVar(const clivalue_t *var, uint8_t value)
```

Comment:
Correct camelCase is `cliOsdProfileSetVar()`.

Reply:
Changed + other instances updated as well.